examples/LoRaWAN/LoRaWAN_End_Device/LoRaWAN_End_Device.ino

```diff
@@ -151,21 +137,25 @@ void loop() {
     Serial.println(F(" Hz"));
   
   } else if(state == RADIOLIB_ERR_RX_TIMEOUT) {
-    Serial.println(F("no downlink!"));
+    Serial.println(F(""));
   
   } else {
     Serial.print(F("failed, code "));
     Serial.println(state);
   }
 
-  // on EEPROM enabled boards, you should save the current session
-  // by calling "saveSession" which allows retrieving the session after reboot or deepsleep
+  // on boards that can save to Flash or EEPROM this saves the session
+  // which allows recall of the session after reboot or deepsleep
   node.saveSession();
   
   // wait before sending another packet
-  uint32_t minimumDelay = 60000;                  // try to send once every minute
-  uint32_t interval = node.timeUntilUplink();     // calculate minimum duty cycle delay (per law!)
+  uint32_t minimumDelay = 300000;                 // try to send once every 3 minutes
+  uint32_t interval = node.timeUntilUplink();     // calculate minimum duty cycle delay (per FUP & law!)
   uint32_t delayMs = max(interval, minimumDelay); // cannot send faster than duty cycle allows
-
+  
+  Serial.print(F("[LoRaWAN] Next uplink in "));
+  Serial.print(delayMs/60);
```

Comment:
This should probably be `delayMs/1000` as pointed out by @ropg.

Reply:
I was trying his coding fast, break things style!